grafana/server.sls

```diff
@@ -14,6 +14,16 @@ grafana_packages:
   - require:
     - pkg: grafana_packages
 
+/etc/grafana/ldap.toml:
```

Comment:
dummy question, any reason to always install ldap.toml even if LDAP auth is not enabled ?